installer/conf/container.conf

```diff
@@ -36,7 +36,7 @@
 #custom_metrics_mdm filter plugin
 <filter mdm.cadvisorperf**>
   type filter_cadvisor2mdm
-  custom_metrics_azure_regions eastus,southcentralus,westcentralus,westus2,southeastasia,northeurope,westeurope,southafricanorth,centralus,northcentralus,eastus2,koreacentral,eastasia,centralindia,uksouth,canadacentral
+  custom_metrics_azure_regions eastus,southcentralus,westcentralus,westus2,southeastasia,northeurope,westeurope,southafricanorth,centralus,northcentralus,eastus2,koreacentral,eastasia,centralindia,uksouth,canadacentral,francecentral,japaneast,australiaeast
```

Comment:
also change in kube.conf (for 2 filter plugins), and then in omsagent.yaml (oms-docker repo) for kube.conf changes

Reply:
updated kube.conf and also created a PR to update oms-docker: https://github.com/microsoft/OMS-docker/pull/341